x-pack/plugins/cases/public/components/user_profiles/translations.ts

```diff
@@ -39,14 +39,14 @@ export const REMOVE_ASSIGNEES = i18n.translate(
   }
 );
 
-export const TOTAL_USERS = (total: number) =>
-  i18n.translate('xpack.cases.userProfile.totalUsers', {
-    defaultMessage: '{total, plural, one {# user} other {# users}}',
-    values: { total },
-  });
-
-export const TOTAL_USERS_ASSIGNED = (total: number) =>
-  i18n.translate('xpack.cases.userProfile.totalUsersAssigned', {
-    defaultMessage: '{total} assigned',
-    values: { total },
-  });
+export const ASSIGNEES = i18n.translate('xpack.cases.assigneesTitle', {
+  defaultMessage: 'Assignees',
+});
+
+export const NO_MATCHING_USERS = i18n.translate('xpack.cases.userProfiles.noMatchingUsers', {
+  defaultMessage: 'No matching users with required access.',
```

Comment:
The `with required access` can be misleading here as it is displayed when running a search for a user with the correct privileges, but who doesn't currently have any cases assigned to them (see also earlier comment about behavior when searching for a user with no cases assigned).

Reply:
Good point, after chatting with Aris and Michael we're going to display all users regardless of whether they've actually been assigned to a case or not.